patches/tModLoader/Terraria/ModLoader/ModLoader.cs

```diff
@@ -176,11 +176,22 @@ private static void Load(CancellationToken token = default)
 			if (e.Data.Contains("contentType") && e.Data["contentType"] is Type contentType)
 				msg += "\n" + Language.GetTextValue("tModLoader.LoadErrorContentType", contentType.FullName);
 
-			Logging.tML.Error(msg, e);
-
-			foreach (var mod in responsibleMods)
+			foreach (var mod in responsibleMods) {
 				DisableMod(mod);
 
+				var dependents = from m in availableMods
+					where m.properties.modReferences.Any(reference => reference.mod.Equals(mod))
+					select m.Name;
+
+				msg += "\n" + Language.GetTextValue("tModLoader.LoadErrorDependentsDisabled", mod,
```

Comment:
Please aggregate all the disabled dependents into one list so the message only gets added once. 

Please also filter to only enabled mods.

Additionally, you will need to write this in a recursive manner, because you may have a series of mod references `B requires C requires A` and if `A` gets disabled, then you have to disable `C` and then `B` as well.